make/modules/java.desktop/Lib.gmk

```diff
@@ -51,11 +52,22 @@ ifeq ($(call isTargetOs, aix), false)
       -DUSE_PLATFORM_MIDI_IN=TRUE \
       #
 
+  ifeq ($(call isTargetOs, bsd), false)
```

Comment:
Is this correct? That adds flags to all the existing platforms but not BSD.

Reply:
That in itself is not the main problem with this part I think. If those flags makes sense for all platforms except BSD's this makes sense. It's less sensible that the added flags are already added on the lines above, regardless of whether isTargetOs returns true or false. I suspect that's a sloppy merge somewhere.

Reply:
Ah, I see, yes in that case I assume this is a merge mistake, and then it makes sense.